site/content/en/latest/tasks/quickstart.md

```diff
@@ -20,7 +20,7 @@ so the `Gateway` resource has an Address associated with it. We recommend using
 Install the Gateway API CRDs and Envoy Gateway:
 
 ```shell
-helm install eg oci://docker.io/envoyproxy/gateway-helm --version {{< helm-version >}} -n envoy-gateway-system --create-namespace
+helm install eg oci://docker.io/envoyproxy/gateway-helm -n envoy-gateway-system --create-namespace
```

Comment:
I don't think we need this. When navigating to versioned install docs you get an actual version vs v0.0.0-latest, e.g. https://gateway.envoyproxy.io/v1.6/install/install-helm/#install-with-helm.

Reply:
I'm fine with setting it to the latest release, as only for helm install.
But if it is described with the notice, I think we can keep the old style, we could improve it by setting something like preview, git tag instead of v0.0.0-latest.